snippets.py

```diff
@@ -659,20 +659,14 @@ def admin_manage_users_handler():
         user_data.append((user.email, user.is_hidden,
                           user.created, weeks_since_snippet))
 
-    # We have to use 'cmp' here since we want ascending in the
-    # primary key and descending in the secondary key, sometimes.
     if sort_by == 'email':
-        user_data.sort(key=cmp_to_key(lambda x, y: cmp(x[0], y[0])))
+        user_data.sort(key=lambda x: x[0])
     elif sort_by == 'creation_time':
-        user_data.sort(key=cmp_to_key(
-            lambda x, y: (-cmp(x[2] or datetime.datetime.min,
-                               y[2] or datetime.datetime.min)
-                          or cmp(x[0], y[0]))))
+        user_data.sort(key=lambda x: x[0])
+        user_data.sort(key=lambda x: x[2] or datetime.datetime.min, reverse=True)
     elif sort_by == 'last_snippet_time':
-        user_data.sort(key=cmp_to_key(
-            lambda x, y: (-cmp(1000 if x[3] is None else x[3],
-                               1000 if y[3] is None else y[3])
-                          or cmp(x[0], y[0]))))
+        user_data.sort(key=lambda x: x[0])
+        user_data.sort(key=lambda x: 1000 if x[3] is None else x[3], reverse=True)
```

Comment:
This one you could do:
```
user_data.sort(key=lambda x: (-1000 if x[3] is None else -x[3], x[0]))
```
though maybe you want to keep it the way it is for parallelism.

Reply:
Yeah, I only did it this way to be consistent with the creation_time branch.  Up to you if you'd prefer the single-pass style.